### PR TITLE
DDF-4211 update Admin UI tool tip

### PR DESCRIPTION
**Pull request by pvargas**
Forwardports #3862 

What does this PR do?
It updates the Admin UI documentation and tool tip for org.codice.ddf.security.filter.login.Session, to explain the logout behavior better.

Who is reviewing it?
@emmberk @oconnormi @mdang8 @peterhuffer @kcover

Select relevant component teams:
@codice/docs
@codice/ui

Ask 2 committers to review/merge the PR and tag them here.
@clockard
@vinamartin

What are the relevant tickets?
DDF-4211

Notes on Review Process
Please see Notes on Review Process for further guidance on requirements for merging and abbreviated reviews.

Review Comment Legend:
✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist.
❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by pvargas** [comment]
Build Now

**Comment by pvargas** [comment]
Formatting error.
Build Now

**Comment by pvargas** [comment]
Build now!

**Comment by oconnormi** [comment]
Build Now

**Comment by pvargas** [comment]
Build now!!

File: platform/platform-configuration/src/main/java/org/codice/ddf/configuration/PlatformUiConfiguration.java [hunk 1]
```diff
@@ -56,7 +56,7 @@
 
   public static final String TIMEOUT_CONFIG_KEY = "timeout";
 
-  private static final int DEFAULT_TIMEOUT_MINUTES = 15;
+  private static final int DEFAULT_TIMEOUT_MINUTES = 31;
```
Comment (clockard) [inline platform/platform-configuration/src/main/java/org/codice/ddf/configuration/PlatformUiConfiguration.java]: This should still be 15.

File: distribution/docs/src/main/resources/content/_reference/_tables/platform.ui.config.adoc [hunk 2]
```diff
@@ -75,8 +75,9 @@
 |Session Timeout
 |timeout
 |Integer
-|Specifies the length of inactivity (in minutes) that will cause a user to be logged out automatically. This value must be 2 minutes or greater, as users are warned when only 1 minute remains. If a value of less than 2 minutes is used, the timeout is set to the default time of 15 minutes.
-|15
+|Specifies the length of inactivity (in minutes) that will cause a user to be logged out automatically. This value must be 2 minutes or greater, as users are warned when only 1 minute remains. If a value of less than 2 minutes is used, the timeout is set to the default time of 31 minutes.
+
+|31
```
Comment (clockard) [inline distribution/docs/src/main/resources/content/_reference/_tables/platform.ui.config.adoc]: This should be 15.

File: distribution/docs/src/main/resources/content/_reference/_tables/Session.adoc [hunk 3]
```diff
@@ -19,8 +19,12 @@
...
 |expirationTime
 |Integer
-|The number of minutes after a session has been inactive that it should be invalidated.
-|31
+|Specifies the length of inactivity (in minutes) between client requests before the servlet container will invalidate the session (this applies to all client sessions).
+This value must be 2 minutes or greater, as users are warned when only 1 minute remains. If a value of less than 2 minutes is used, the timeout is set to the default time of 15 minutes.
+
+See also: <<_ddf.platform.ui.config,Platform UI Config>>.
+
+|15
```
Comment (clockard) [inline distribution/docs/src/main/resources/content/_reference/_tables/Session.adoc]: This should be 31

File: distribution/docs/src/main/resources/content/_reference/_tables/Session.adoc [hunk 4]
```diff
@@ -19,8 +19,12 @@
 |Session Timeout (in minutes)
 |expirationTime
 |Integer
-|The number of minutes after a session has been inactive that it should be invalidated.
-|31
+|Specifies the length of inactivity (in minutes) between client requests before the servlet container will invalidate the session (this applies to all client sessions).
+This value must be 2 minutes or greater, as users are warned when only 1 minute remains. If a value of less than 2 minutes is used, the timeout is set to the default time of 15 minutes.
```
Comment (clockard) [inline distribution/docs/src/main/resources/content/_reference/_tables/Session.adoc]: Should be 31 minutes

File: platform/platform-configuration/src/main/resources/OSGI-INF/metatype/metatype.xml [hunk 5]
```diff
@@ -66,7 +66,7 @@
         <AD
                 description="Specifies the length of inactivity (in minutes) that will cause a user to be logged out automatically. This value must be 2 minutes or greater, as users are warned when only 1 minute remains. If a value of less than 2 minutes is used, the timeout is set to the default time of 15 minutes."
                 name="Session Timeout" id="timeout" required="true" type="Integer"
-                default="15"
+                default="31"
```
Comment (clockard) [inline platform/platform-configuration/src/main/resources/OSGI-INF/metatype/metatype.xml]: Should be 15

File: platform/security/filter/security-filter-login/src/main/java/org/codice/ddf/security/filter/login/LoginFilter.java [hunk 6]
```diff
@@ -100,6 +100,8 @@
 
   private static final String DDF_AUTHENTICATION_TOKEN = "ddf.security.token";
 
+  private static final int DEFAULT_EXPIRATION_TIME = 15;
```
Comment (clockard) [inline platform/security/filter/security-filter-login/src/main/java/org/codice/ddf/security/filter/login/LoginFilter.java]: Should be 31

**Comment by pvargas** [comment]
Build now.

**Pull request merged**